src/CalcManager/Ratpack/num.cpp

```diff
@@ -192,9 +192,9 @@ void _addnum( PNUMBER *pa, PNUMBER b, uint32_t radix)
 //
 //----------------------------------------------------------------------------
 
-void _mulnum( PNUMBER *pa, PNUMBER b, uint32_t radix);
+void _mulnum( _Inout_ PNUMBER *pa, _In_ _Const_ PNUMBER b, uint32_t radix);
```

Comment:
What's the purpose of `_Const_` here? What benefit does it bring over `const`?

Reply:
I don't want to change the interface of _mulnum, and (IIRC) it does actually modify it a few times within the function, before returning it to the original state.

Reply:
That makes no sense. Either it is const, in which case it should be marked so with `const` or it isn't and you shouldn't try faking it.

Reply:
I think you're correct here.